packages/typespec-autorest-canonical/README.md

```diff
@@ -0,0 +1,84 @@
+# @azure-tools/typespec-autorest-canonical
+
+TypeSpec library for emitting canonical version swagger
+
+## Install
+
+```bash
+npm install @azure-tools/typespec-autorest-canonical
+```
+
+## Emitter
+
+### Usage
+
+1. Via the command line
+
+```bash
+tsp compile . --emit=@azure-tools/typespec-autorest-canonical
+```
+
+2. Via the config
+
+```yaml
+emit:
+  - "@azure-tools/typespec-autorest-canonical"
+```
+
+### Emitter options
+
+#### `output-file`
+
+**Type:** `string`
+
+Name of the output file.
+Output file will interpolate the following values:
+
+- service-name: Name of the service if multiple
+- azure-resource-provider-folder: Value of the azure-resource-provider-folder option
+
+Default: `{azure-resource-provider-folder}/{service-name}/canonical/openapi.json`
```

Comment:
I think it makes sense that this is alongside the versioned specs, this location seems to not interfere with the ci, do you have concerns about this structure?

Reply:
More my concern in the meeting about commiting the file all together.

Ideally we would only have the typespec in those repos I think and whatever needs to depend on it can compile in demand. 

But if that's really the way to go then yeah this structure is fine.